cli/cmd/cli_state.go

```diff
@@ -231,6 +231,7 @@ func (c *cliState) NewClient() error {
 		api.WithLogLevel(c.Log.Level().CapitalString()),
 		api.WithSubaccount(c.Subaccount),
 		api.WithApiKeys(c.KeyID, c.Secret),
+		api.WithTransport(&http.Transport{TLSHandshakeTimeout: time.Second * 63}),
```

Comment:
Could we move the default to the SDK level, that is, inside the `api` package instead of setting defaults for CLI, Terraform, etc.